homeassistant/components/harmony/util.py

```diff
@@ -11,7 +11,7 @@ def find_unique_id_for_remote(harmony: HarmonyAPI):
     """Find the unique id for both websocket and xmpp clients."""
     websocket_unique_id = harmony.hub_config.info.get("activeRemoteId")
     if websocket_unique_id is not None:
-        return str(websocket_unique_id)
+        return websocket_unique_id
```

Comment:
@bdraco Is there a chance that this isn't a string? In that case we should make sure it is a string.

Reply:
The web socket unique id is a numeric so casting it to a string make sense as long as we backport that change to 0.108 so it doesn’t change between versions